release-notes/v3.29.0-release-notes.md

```diff
@@ -0,0 +1,96 @@
+28 Oct 2024
+
+#### AdminNetworkPolicy support
+
+- Add support for AdminNetworkPolicy egress network CIDRs. [calico 9303](https://github.com/projectcalico/calico/pull/9303) (@mazdakn)
+- Allow users to specify the default action in a tier. [calico 9245](https://github.com/projectcalico/calico/pull/9245) (@mazdakn)
+- Add support for the core functionalities of the  new k8s AdminNetworkPolicy API. [calico 9206](https://github.com/projectcalico/calico/pull/9206) (@mazdakn)
+- Tiers: A tier's order is by default 1,000,000. A nil order similarly is treated as 1M. [calico 9189](https://github.com/projectcalico/calico/pull/9189) (@mazdakn)
+- Add tiered policy support [calico 9085](https://github.com/projectcalico/calico/pull/9085) (@mazdakn)
+
+#### nftables dataplane (tech-preview)
+
+- Tech-preview support for nftables dataplane and nftables kube-proxy compatibility [calico 8780](https://github.com/projectcalico/calico/pull/8780) (@caseydavenport)
+
+#### Bug fixes
+
+- Fix spurious warning about unexpected inserted rules. [calico 9397](https://github.com/projectcalico/calico/pull/9397) (@fasaxc)
+- Fixed memory leak in BPF endpoint manager. [calico 9309](https://github.com/projectcalico/calico/pull/9309) (@tomastigera)
+- fix image in flannel migration manifest [calico 9265](https://github.com/projectcalico/calico/pull/9265) (@radTuti)
```

Comment:
Would be good to clean up sentence casing throughout, so sentences start with an upper case letter.